### ISSUE TITLE
[feature request] Caffe2 model to PyTorch model

### ISSUE THREAD

**Issue by Oktai15** [pytorch/pytorch#7276]
Do you think about converting Caffe2 model to PyTorch model for continue changing model, e.g fine-tuning, transfer-learning?

According to new feature of PyTorch 1.0, it seems possible, isn't it?

**Comment by soumith** [comment]
at the moment this is not planned as such (to create a convertor for caffe2 model to pytorch model). it's not listed in the 1.0 roadmap either.
However, you can import both caffe2 and pytorch into the same python, and get the output of caffe2 and send it to your pytorch model, and similarly for the gradients (get them from pytorch backward and send them to caffe2)
